src/woo_publications/publications/api/serializers.py

```diff
@@ -72,6 +74,33 @@ class Meta:  # pyright: ignore
             "laatst_gewijzigd_datum",
             "bestandsdelen",
         )
+        extra_kwargs = {
+            "uuid": {
+                "read_only": True,
+            },
+            "publicatiestatus": {
+                "help_text": _(
+                    "\n**Disclaimer**: you can't create a {} document."
+                ).format(PublicationStatusOptions.revoked.label.lower())
+            },
+        }
+
+    def validate(self, attrs):
+        self.instance: Document
```

Comment:
I'd declare this outside of the method, and perhaps it can be passed as a generic too, I'll experiment with it a little.

either way - `instance` can also be None, I think, but you aren't using it in this method so I don't think it's even necessary in the first place.

```suggestion
```